### ISSUE TITLE
Not Connecting from our server

### ISSUE THREAD

**Issue by TheEmberLeader** [barbushin/php-console#91]
We have php console installed via composer on  a standalone domain on our server (no wordpress or similar application).  Basically we have an app that is making calls to a php file, and in that php file we want to initialize the php console.  However, we can't get it to send the request to authenticate and establish a connection.  Right now we're testing our app in chrome from desktop (the app is currently web languages)
Here's our initialization code.

require __DIR__ . '/vendor/autoload.php';	
	$connector = PhpConsole\Connector::getInstance();
	PhpConsole\Helper::register();
	$connector->setPassword('123456', false);
	$connector->getDebugDispatcher()->dispatchDebug('ok');
	PC::debug("our stuff");

What are we doing wrong?

**Comment by TheEmberLeader** [comment]
Note: we're not getting the PC::debug message at all, but there are no errors in the error_log.

**Comment by barbushin** [comment]
And does it works on http://php-console.com/instance/examples/features/debug_vars.php?

**Comment by TheEmberLeader** [comment]
Yes, it works there. Everything is up to date, and we're using the standard version of Chrome.

**Comment by barbushin** [comment]
What is URL of your local server? What is web-server?

**Comment by barbushin** [comment]
Do you see authorization icon?

**Comment by TheEmberLeader** [comment]
Url is phonebook.mp0studios.com
Web server is Linux, CentOS 6.

We don't see that link.

**Comment by barbushin** [comment]
I need URL to the PHP script with activated php-console :) There must be icon like this http://i.imgur.com/RrE80zy.png

**Comment by TheEmberLeader** [comment]
phonebook.mp0studios.com/pb.php

We don't see that icon when we go to the script.

**Comment by barbushin** [comment]
It works for me http://i.imgur.com/SXldzCi.png What is your OS and Chrome version?

**Comment by TheEmberLeader** [comment]
Okay, we checked the console there and you're right it works, but it doesn't work when we call the php script via Ajax.

**Comment by TheEmberLeader** [comment]
Os is windows 8.1.  Chrome is 44.0.2403.89 m. Php console is 3.0.35

**Comment by tarstarkes** [comment]
We attempted to instantiate the pc debugger inside of one of our ajax calls, one that we know is being called on initialization; we also attempted to simply call PC::debug within the ajax call with all of the initialization still globally declared at the top of our file but all to to no avail.

**Comment by tarstarkes** [comment]
the main file we load is an html file which calls all of this on the server, however most of the templates and javascript code is hosted locally (that includes the html file mentioned previously).

**Comment by TheEmberLeader** [comment]
Okay, so we tried uploading all our files (index.html, main.js, style.css) to the server where the pb.php file is that initializes the php console.  It works in that environment.  So, the fact that our test files are local to our desktop is the issue.  While the files were on the server, i changed our ajax call between jsonp and json, neither broke the console there.  So, for some reason, if our html and js files are local, calling a php file on a server, then the php console won't initialize.
Hope this helps you find a solution.

**Comment by TheEmberLeader** [comment]
We have an html file and a js file that I can send you to download and run locally (it'll do the ajax request to a php file on our server that initializes the php console and tries to do a PC::debug call.)  We don't see the php console icon or get any output to the console.

**Comment by TheEmberLeader** [comment]
Also, just to check, I put our test files all on a local WAMP server to see if that worked, and it still will not work in that context either.  I'm at a loss for what's going on.  We really need to be able to test and debug locally, as that is the closest to how the app will be running when it's live.

**Comment by barbushin** [comment]
@TheEmberLeader Please send this files on barbushin@gmail.com

**Comment by barbushin** [comment]
Here is one thing: php-console extension listener becomes activated for some tab when `php-console-server` cookie is detected in the response headers of this tab. That means that it will handle only requests from some domain/subdomain.

There is some very tricky way how I can fix it, but it will take some time. 

What I can recommend for now, is just to put HTML file on the same domain as PHP file.

**Comment by barbushin** [comment]
Okay, so that issue is just duplicate of https://github.com/barbushin/php-console/issues/54

If you're interested when it will be fixes, just subscribe on https://github.com/barbushin/php-console/issues/54 and you'll get notification.

**Comment by TheEmberLeader** [comment]
Thanks, I wasn't aware of this issue.  Though I'm curious why it didn't work on a local WAMP server either.  Anyway, in-case it helps, I sent the files.
